server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/ParquetResolver.java

```
@@ -356,4 +461,20 @@ private OneField resolveField(Group group, int columnIndex) {
         }
         return field;
     }
+
+    /**
+     * Returns server configuration value for pxf.parquet.write.decimal.overflow.
+     *
+     * @param configuration contains server configuration properties
+     * @return String value for pxf.parquet.write.decimal.overflow. Must be "error", "round" or "ignore"
+     */
+    public String parseDecimalOverflowOption(Configuration configuration) {
+        String decimalOverflowOption = configuration.get(PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_PROPERTY_NAME, "round");
+        if (StringUtils.equalsIgnoreCase(PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_OPTION_IGNORE, decimalOverflowOption)) {
+            return PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_OPTION_IGNORE;
+        } else if (StringUtils.equalsIgnoreCase(PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_OPTION_ERROR, decimalOverflowOption)) {
+            return PXF_PARQUET_WRITE_DECIMAL_OVERFLOW_OPTION_ERROR;
+        }
```

we need to analyze ROUND and error out if the property has been set to anything else.

---

Do we need to error out if ROUND is not set to "round"? Or we treat anything else as "round" instead of erroring out?